### PR TITLE
Add a scraper for https://streetkitchen.hu

### PR DESCRIPTION
**Pull request by mojito317**
This version supports another big Hungarian recipe page: https://streetkitchen.hu.

### CODE REVIEW

File: recipe_scrapers/streetkitchen.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        for ingredient in ingredient_group:
+            ingredients.append(normalize_string(ingredient.get_text()).strip())
+
+        return ingredients
+
+    def instructions(self):
+        instructions = self.soup.find("div", {"class": "the-content-div"}).findAll("p")[
+            :-1
+        ]  # the last paragraph is advertisement, not instructions
```
Comment (bfcarpio) [inline recipe_scrapers/streetkitchen.py:37]: Thank you for this comment! That's good documentation.

**Review by bfcarpio — CHANGES_REQUESTED** [review]
The code you have here looks good, but it seems you haven't added it to module pool or the list on the README. Once you do that, I'll (we'll) merge it in!

You can see [this example](https://github.com/hhursev/recipe-scrapers/pull/176/files) for all the changes that are needed for a scraper PR.

**Comment by mojito317** [comment]
> The code you have here looks good, but it seems you haven't added it to module pool or the list on the README.

@bfcarpio, thanks! I fixed it in 5136625.

**Review by bfcarpio — APPROVED** [review]

**Comment by bfcarpio** [comment]
Thanks for the updates! Merging

**Pull request merged**